### PR TITLE
Global Components : Header and Navigation Bar done 

### PR DESCRIPTION
**Pull request by lihuicham**
Completed Header and Navigation Bar. 
Active class added.

### CODE REVIEW

**Review by xbrianlong — CHANGES_REQUESTED** [review]
Generally looks good except for a few design intricacies.

Change merge target to dev branch.

Things missing that needed to be added in the future:
-Clicking avatar goes to user profile page/ modal
-Expand Map, Share, Edit functionalities from ItineraryHeader
-Delete functionality + clicking brings up LocationCard functionality from DestinationItem

File: src/components/ItineraryPage/ItineraryHeader.vue
[the remark this answers is not in the document]
Comment (xbrianlong) [reply src/components/ItineraryPage/ItineraryHeader.vue:1]: I realised u push a separate PR for itinerary page so u can ignore this if u fixed it in the new PR. Just fix the navbar problems first then i can merge this branch first.

[the remark this answers is not in the document]
Comment (lihuicham) [reply src/components/ItineraryPage/ItineraryHeader.vue:1]: yes, it is fixed in another PR for itinerary page already ! thanks !

[the remark this answers is not in the document]
Comment (lihuicham) [reply src/components/ItineraryPage/ItineraryHeader.vue:1]: for this pull request, can just look at the header and nav bar - the itinerary page design is in another PR. but, yes, i am working and will continue work on the interactivity of the itinerary page.

File: src/components/GlobalComponents/NavigationBar.vue
[the remark this answers is not in the document]
Comment (lihuicham) [reply src/components/GlobalComponents/NavigationBar.vue:1]: okay, got it ! fixed alr

**Pull request merged**